src/main/scala/edu/cmu/cs/obsidian/codegen/yulAST.scala

```diff
@@ -129,20 +142,24 @@ case class Block (statements: Seq[YulStatement]) extends YulStatement {
     HexLiteral = 'hex' ('"' ([0-9a-fA-F]{2})* '"' | '\'' ([0-9a-fA-F]{2})* '\'')
     StringLiteral = '"' ([^"\r\n\\] | '\\' .)* '"'
  */
-case class Code (block: Block) extends YulAST
-case class Data (name: String, hex: Seq[HexLiteral], str: Seq[StringLiteral]) extends YulAST
-case class HexLiteral (content: String) extends YulAST
-case class StringLiteral (content: String) extends YulAST
-case class YulObject (name: String, code: Code, subObjects: Seq[YulObject], data: Seq[Data]) extends YulAST {
+case class Code(block: Block) extends YulAST
+
+case class Data(name: String, hex: Seq[HexLiteral], str: Seq[StringLiteral]) extends YulAST
+
+case class HexLiteral(content: String) extends YulAST
+
+case class StringLiteral(content: String) extends YulAST
+
+case class YulObject(name: String, code: Code, subs: Seq[YulObject], data: Seq[Data]) extends YulAST {
```

Comment:
Please expand "subs."

Reply:
(it reads like "substitutions" to me, and we're just not that short on space here).

Reply:
oh, yeah, good catch!